Libraries/IonAuth.php

```diff
@@ -0,0 +1,440 @@
+<?php namespace IonAuth\Libraries;
+
+/**
+ * Name:    Ion Auth
+ * Author:  Ben Edmunds
+ *           ben.edmunds@gmail.com
+ *           @benedmunds
+ *
+ * Added Awesomeness: Phil Sturgeon
+ *
+ * Created:  10.01.2009
+ *
+ * Description:  Modified auth system based on redux_auth with extensive customization. This is basically what Redux Auth 2 should be.
+ * Original Author name has been kept but that does not mean that the method has not been modified.
+ *
+ * Requirements: PHP7.1 or above
+ *
+ * @package    CodeIgniter-Ion-Auth
+ * @author     Ben Edmunds
+ * @license    https://opensource.org/licenses/MIT	MIT License
+ * @link       http://github.com/benedmunds/CodeIgniter-Ion-Auth
+ * @filesource
+ */
+
+/**
+ * Class IonAuth
+ */
+class IonAuth
+{
+	/**
+	 * @var \IonAuth\Config\IonAuth
+	 */
+	private $config;
+
+	/**
+	 *
+	 * @var \IonAuth\Models\IonAuthModel
+	 */
+	private $ionAuthModel;
+
+	/**
+	 * Extra where
+	 *
+	 * @var array
+	 **/
+	public $_extra_where = [];
```

Comment:
Camelcase and set proper visibility instead of using the old CI underscore format for protected/private vars
```suggestion
	protected $extraWhere = [];
```

Will need to double check for any uses of this to update as well